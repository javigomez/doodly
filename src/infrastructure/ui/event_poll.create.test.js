import React from 'react'
import EventPollCreate from './event_poll.create'
import { shallow } from 'enzyme'

it('Event Poll Creation Page renders without crashing', () => {
  const component = shallow(<EventPollCreate />)
  expect(component.exists()).toEqual(true)
  expect(component.getElements()).toMatchSnapshot();
})

