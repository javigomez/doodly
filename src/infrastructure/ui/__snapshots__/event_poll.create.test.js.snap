// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Poll Creation Page renders without crashing 1`] = `
Array [
  <div>
    <h1>
        Create a Event Poll
    </h1>
    <form
        method="post"
        name="message"
    >
        <label
            htmlFor="title"
        >
            Event title
        </label>
        <input
            id="title"
            type="text"
        />
        <input
            type="submit"
            value="Submit"
        />
    </form>
</div>,
]
`;
